src/main/java/org/dave/compactmachines3/misc/Vec3d2f.java

```diff
@@ -0,0 +1,85 @@
+package org.dave.compactmachines3.misc;
```

Comment:
Remove this; we should be preferring Vector3d and an optional Vector2f instead of making this wrapper class in 1.12.